string-manipulation-3/src/is-anagram.js

```diff
@@ -0,0 +1,15 @@
+/* exported isAnagram */
+// define function
+function isAnagram(firstString, secondString) {
+  // make firstString string into sorted string without spaces
+  firstString = firstString.split(' ').join('').split('').sort().join('');
+  // make secondString string into sorted string without spaces
+  secondString = secondString.split(' ').join('').split('').sort().join('');
```

Comment:
It may not look too bad, but this solution has _**5**_ loops on line 5 AND line 7. That is a lot of unnecessary complexity. This problem is doable with just one loop, see if you can figure out how 🤓